src/components/Form/ForeignBornDocuments/ForeignBornDocuments.jsx

```diff
@@ -290,6 +290,9 @@ export default class ForeignBornDocuments extends ValidationElement {
             label="Document Number"
             {...this.props.DocumentNumber}
             className="foreign-born-document-number"
+            maxlength="30"
+            pattern="^[a-zA-Z0-9]*$"
```

Comment:
Since this is repeated in a number of places over a few files, lets make this a constant somewhere in the app, and import it to the files we need it in. This way, if we have to make any adjustments to the regex, we wont have to change it in a bunch of places.